src/sentry/backup/dependencies.py

```diff
@@ -95,28 +95,23 @@ def __repr__(self) -> str:
 #
 # TODO(getsentry/team-ospo#190): We should find a better way to store this information than a magic
 # list in this file. We should probably make a field (or method?) on `BaseModel` instead.
-@unique
-class RelocationRootModels(Enum):
```

Comment:
in mypy 1.14.0 enum typing has [changed to adhere to the new specification](https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members) forbidding annotations on enum members.  but it's a bit of a zugzwang as removing the annotation on `Excluded` makes mypy request an annotation for the empty list

but turns out none of this is required as an enum as it's only used to seed the "root" models.  no idea why this was written as an enum when it very much isn't one